README.md

```diff
@@ -5,28 +5,45 @@
 [![Documentation](https://docs.rs/vortex-array/badge.svg)](https://docs.rs/vortex-array)
 [![PyPI - Python Version](https://img.shields.io/pypi/pyversions/vortex-array)](https://pypi.org/project/vortex-array/)
 
-Vortex is an Apache Arrow-compatible toolkit for working with compressed array data. We are using Vortex to develop a
-next-generation columnar file format for multidimensional arrays.
+Vortex is a toolkit for working with compressed Apache Arrow arrays in-memory, on-disk, and over-the-wire.
+
+Vortex is designed to be to columnar file formats what Apache DataFusion is to query engines (or, analogously,
+what LLVM + Clang are to compilers): a highly extensible & extremely performant *framework* for building a modern
+columnar file format, with a state-of-the-art, "batteries included" reference implementation that we're tentatively
+calling "Vinyl".[1]
+
+Vinyl is an aspiring successor to Apache Parquet, with dramatically faster random access reads (100-200x faster)
+and scans (2-10x faster), while preserving approximately the same compression ratio and write throughput. It will also support very wide
+tables (at least 10s of thousands of columns) and (in the future) on-device decompression on GPUs.
 
 > [!CAUTION]
-> This library is still under rapid development and is very much a work in progress!
+> This library is still under rapid development and is a work in progress!
 >
-> Some key features are not yet implemented, the API will almost certainly change in breaking ways, and we cannot
-> yet guarantee correctness in all cases.
+> Some key features are not yet implemented, both the API and the serialized format are likely to change in breaking ways,
+> and we cannot yet guarantee correctness in all cases.
 
-The major components of Vortex are (will be!):
+The major components of Vortex are:
 
 * **Logical Types** - a schema definition that makes no assertions about physical layout.
-* **Encodings** - a pluggable set of physical layouts. Vortex ships with several state-of-the-art lightweight
-  compression codecs that have the potential to support GPU decompression.
-* **Compression** - recursive compression based on stratified samples of the input.
-* **Compute** - basic compute kernels that can operate over compressed data. Note that Vortex does not intend to become
-  a full-fledged compute engine, but rather to provide the ability to implement basic compute operations as may be
-  required for efficient scanning & pushdown operations.
+* **Zero-Copy to Arrow** - "canonicalized" (i.e., fully decompressed) Vortex arrays can be zero-copy converted to/from Apache Arrow arrays.
```

Comment:
These read more like features than components now. Maybe change the lead-in sentence?